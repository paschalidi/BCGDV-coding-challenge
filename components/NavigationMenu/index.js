/**
 *
 * NavigationMenu
 *
 */
import React from 'react';
import { Button } from 'semantic-ui-react';
import InternalLink from '../InternalLink';


const NavigationMenu = () =>
  <div>
    <InternalLink href={`/`}>
      <Button secondary fluid>Back</Button>
    </InternalLink>
  </div>;

export default NavigationMenu;