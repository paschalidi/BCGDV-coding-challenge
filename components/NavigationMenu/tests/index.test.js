/* eslint-env jest */
import { shallow } from 'enzyme';
import React from 'react';
import NavigationMenu from '../index';

describe('Snapshot Testing', () => {
  it('NavigationMenu', () => {
    const component = shallow(<NavigationMenu />);
    expect(component).toMatchSnapshot();
  });
});

