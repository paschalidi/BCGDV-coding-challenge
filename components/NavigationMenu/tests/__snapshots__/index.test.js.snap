// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing NavigationMenu 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavigationMenu />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <InternalLink
        href="/"
      >
        <Button
          as="button"
          fluid={true}
          role="button"
          secondary={true}
        >
          Back
        </Button>
      </InternalLink>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <Button
          as="button"
          fluid={true}
          role="button"
          secondary={true}
        >
          Back
        </Button>,
        "href": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "as": "button",
          "children": "Back",
          "fluid": true,
          "role": "button",
          "secondary": true,
        },
        "ref": null,
        "rendered": "Back",
        "type": [Function],
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <InternalLink
          href="/"
        >
          <Button
            as="button"
            fluid={true}
            role="button"
            secondary={true}
          >
            Back
          </Button>
        </InternalLink>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <Button
            as="button"
            fluid={true}
            role="button"
            secondary={true}
          >
            Back
          </Button>,
          "href": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "as": "button",
            "children": "Back",
            "fluid": true,
            "role": "button",
            "secondary": true,
          },
          "ref": null,
          "rendered": "Back",
          "type": [Function],
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
